### PR TITLE
Fix code smells

### PR DESCRIPTION
**Pull request by ipaunov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
   "dependencies": {
     "@sap-cloud-sdk/connectivity": "^3.5.0",
     "@sap-cloud-sdk/http-client": "^3.5.0",
-    "@sap-cloud-sdk/util": "^3.5.0"
+    "@sap-cloud-sdk/util": "^3.5.0",
+    "@sap/xssec": "^3.6.0",
```
Comment (sidakphull) [inline package.json:22]: why are these added?

File: package-lock.json
Comment (sidakphull) [inline package-lock.json:1]: should we remove this? @simeonPetkov96

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,4 +9,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
...
 
-- Initial release
\ No newline at end of file
+- Initial release
+
+## Version 0.2.0
+
+### Added
+
+- Fixed code smells and Dependabot alerts
```
Comment (sidakphull) [inline CHANGELOG.md:18]: shouldn't this go in `Fixed` section?

Comment (ipaunov) [reply CHANGELOG.md:18]: done

File: package.json (hunk 1 above)
Comment (sidakphull) [reply package.json:22]: I think you should update the `sap-cloud-sdk` libraries. The dependabot alerts might be coming because one of the `sap-cloud-sdk` library uses these  `sap/xssec`, `axios` and `follow-redirects`.

Comment (ipaunov) [reply package.json]: done

**Review by RamIndia — APPROVED** [review]
Looks Good.
